runtime/doc/various.txt

```diff
@@ -325,7 +324,6 @@ N  *+find_in_path*	include file searches: |[I|, |:isearch|,
 			|CTRL-W_CTRL-I|, |:checkpath|, etc.
 N  *+folding*		|folding|
 N  *+gettext*		message translations |multi-lang|
-   *+GUI_GTK*		Unix only: GTK+ |GUI|
```

Comment:
Not sure why `make -C runtime/doc` isn't showing any errors, but this is still referenced in `options.txt` in three separate options, which could probably be removed given they're no-ops.